samples/Hangfire.Core.MvcApplication/Startup.cs

```diff
@@ -30,27 +28,29 @@ public void ConfigureServices(IServiceCollection services)
         {
             services.Configure<CookiePolicyOptions>(options =>
             {
-                // This lambda determines whether user consent for non-essential cookies is needed for a given request.
+                // This lambda determines whether user consent for non-essential cookies is needed
+                // for a given request.
                 options.CheckConsentNeeded = context => true;
                 options.MinimumSameSitePolicy = SameSiteMode.None;
             });
 
             services.AddHangfire(config =>
             {
                 config.UseMemoryStorage();
-//                config.UseSqlServerStorage(Configuration.GetConnectionString("DefaultConnection"), new SqlServerStorageOptions
-//                {
-//                    SlidingInvisibilityTimeout = TimeSpan.FromMinutes(5), // To enable Sliding invisibility fetching
-//                    CommandBatchMaxTimeout = TimeSpan.FromMinutes(5), // To enable command pipelining
-//                    QueuePollInterval = TimeSpan.FromTicks(1) // To reduce processing delays to minimum
-//                });
+                // config.UseSqlServerStorage(Configuration.GetConnectionString("DefaultConnection"),
+                // new SqlServerStorageOptions { SlidingInvisibilityTimeout =
+                // TimeSpan.FromMinutes(5), // To enable Sliding invisibility fetching
+                // CommandBatchMaxTimeout = TimeSpan.FromMinutes(5), // To enable command pipelining
+                // QueuePollInterval = TimeSpan.FromTicks(1) // To reduce processing delays to
+                // minimum });
```

Comment:
Since this is a sample should we be changing the whitespace here?

Reply:
rolling it back